blueprints/app/files/package.json

```diff
@@ -72,7 +72,7 @@
     "@typescript-eslint/parser": "^6.21.0<% } %>",
```

Comment:
v7.* (of parser + eslint-plugin) may be required for newer TS features https://github.com/typescript-eslint/typescript-eslint/releases?page=3